src/openloco/window.cpp

```diff
@@ -726,6 +766,25 @@ namespace openloco::ui
         }
     }
 
+    void window::call_draw_scroll(gfx::drawpixelinfo_t* dpi, uint32_t scrollIndex)
+    {
+        if (event_handlers->draw_scroll != nullptr)
```

Comment:
Just escape early?

Reply:
Sure... I was just sticking to the convention you had already set up. I'll change it.